build.sbt

```diff
@@ -30,12 +35,18 @@ val dep = {
 val Integration = config("it") extend Test
 
 val root = project.in(file("."))
-    .configs(Integration)
-    .settings(commonSettings: _*)
-    .settings(libraryDependencies ++= dep)
-    .settings(inConfig(Integration)(Defaults.testSettings) : _*)
+  .configs(Integration)
+  .settings(commonSettings: _*)
+  .settings(libraryDependencies ++= dep)
+  .settings(inConfig(Integration)(Defaults.testSettings) : _*)
 
 scalacOptions := Seq("-unchecked", "-deprecation", "-feature")
 
 (scalastyleConfig in Test) := baseDirectory.value / "scalastyle-test-config.xml"
-scalastyleSources in Test ++= {(unmanagedSourceDirectories in Integration).value}
\ No newline at end of file
+scalastyleSources in Test ++= {(unmanagedSourceDirectories in Integration).value}
+
+mappings in Universal ++= (resourceDirectory in Compile).value.listFiles()
```

Comment:
Files in a src/universal/resources/ should get dropped into the conf folder automagically?
(Have a look at https://github.com/mcsherrylabs/sss.asado/tree/master/sss.asado-nobu/src/universal/conf)